src/Widgets/TerminalView.vala

```diff
@@ -165,6 +165,18 @@ public class Terminal.TerminalView : Gtk.Box {
         tab_view.selected_page = target;
     }
 
+    public void cycle_tabs (Hdy.NavigationDirection direction) {
+        var pos = (int) tab_view.get_page_position (selected_page);
```

Comment:
Is there any reason you explicitly cast to int here? The return type of `Hdy.TabView.get_page_position ()` seems to be int: https://valadoc.org/libhandy-1/Hdy.TabView.get_page_position.html